### ISSUE TITLE
Keyboard Lock

### ISSUE THREAD

**Issue by lgarron** [chromium/permission.site#46]
Intent to ship: https://groups.google.com/a/chromium.org/forum/?utm_medium=email&utm_source=footer#!msg/blink-dev/isXS3f3Tqo8/GPIQi_8oCAAJ
Spec: https://w3c.github.io/keyboard-lock/
Chrome design doc: https://docs.google.com/document/d/1z8_VDDj5qrixHxXYeYrotJna_lsBtpzfL7uFKj7F3sA/edit#

Simple use case:

    navigator.keyboard.lock();

However, this does not appear to be available in Chrome Canary yet, even with the `#system-keyboard-lock` and `#keyboard-lock-api` flags enabled.

**Comment by lgarron** [comment]
> Currently this is only available in fullscreen IIRC.

Indeed, that seems to work! The UX seems to be a bit finnicky (multiple things popping up, no none of them specifically mentioning keyboard lock).

Are there simple instructions about the UI we could incorporate so that users know how to get out of it after pressing the button?

**Comment by dominickng** [comment]
I think what's meant to happen is that the usual fullscreen bubble says "Hold Esc to exit" instead of "Press Esc to exit". And that message comes back if we detect the user trying to exit but failing (pressing Esc a lot for instance). I haven't had a chance to play with it yet though :)
